Corona-Warn-App/src/main/res/layout/include_interoperability.xml

```diff
@@ -0,0 +1,228 @@
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/label_interoperability_subtitle2" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+            <TextView
+                android:id="@+id/country_header_description"
+                style="@style/headline6"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_normal"
+                android:accessibilityHeading="true"
+                android:text="@{countryListTitle}"
+                android:visibility="@{FormatterHelper.formatVisibilityText(countryListTitle)}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/interoperability_header" />
+
+            <de.rki.coronawarnapp.ui.view.CountrySelectionList
+                android:id="@+id/countryList"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_normal"
+                app:countryList="@{interopViewModel.allCountries}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/country_header_description" />
+
+            <TextView
+                android:id="@+id/label_country_selection_info"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="@dimen/spacing_normal"
+                android:accessibilityHeading="true"
+                android:text="@{countryListHint}"
+                android:visibility="@{FormatterHelper.formatVisibilityText(countryListHint)}"
+                app:layout_constraintTop_toBottomOf="@+id/countryList" />
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                style="@style/cardTracing"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_normal"
+                android:padding="@dimen/card_padding"
+                android:visibility="@{FormatterHelper.formatVisibility(showFooter)}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/label_country_selection_info">
+                app:layout_constraintTop_toBottomOf="@+id/countryList"/>
+
+                <TextView
+                    android:id="@+id/country_footer_title"
```

Comment:
Would make `interoperability_footer_title` sense or is this `country` needed here?